code/modules/antagonists/traitor/objectives/final_objective/supermatter_cascade.dm

```diff
@@ -20,13 +20,13 @@
 	replace_in_name("%AREA%", initial(dest_crystal_area_pickup.name))
 	return TRUE
 
-/datum/traitor_objective/final/end_reality/generate_ui_buttons(mob/user)
+/datum/traitor_objective/final/supermatter_cascade/generate_ui_buttons(mob/user)
 	var/list/buttons = list()
 	if(!sent_crystal)
-		buttons += add_ui_button("", "Pressing this will call down a pod with the universe ending kit.", "biohazard", "destabilizing_crystal")
+		buttons += add_ui_button("", "Pressing this will call down a pod with the supermatter cascade kit.", "biohazard", "destabilizing_crystal")
```

Comment:
what's the "biohazard" argument here for?

Reply:
I think it's a reference either to the icon file or to the background to use for the button.